config/config.go

```diff
@@ -9,10 +9,9 @@ import (
 	"strings"
 	"time"
 
-	yaml "gopkg.in/yaml.v2"
-
 	"github.com/kiali/kiali/config/security"
 	"github.com/kiali/kiali/log"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
nitpick - this is wrong - it was correct the way it was. Third-party imports, followed by blank line, followed by kiali imports.